src/dispatch/case/flows.py

```diff
@@ -197,6 +199,7 @@ def case_new_create_flow(
     conversation_target: str = None,
     service_id: int = None,
     db_session=None,
+    create_resources: bool = True,
```

Comment:
Should we make this configurable in the UI at case type level?

Reply:
Yeah, I can add in a follow up PR.